src/Stryker.Core/Stryker.Core/Mutators/Mutator.cs

```diff
@@ -33,7 +33,7 @@ public enum Mutator
         String,
         [Description("Bitwise operators")]
         Bitwise,
-        [Description("Array initializer")]
+        [Description("Initializer")]
```

Comment:
This is a breaking change... We need to find a way to keep this backwards compatible